composer.json

```diff
@@ -18,11 +18,11 @@
         "friendsofphp/php-cs-fixer": "^3.89",
         "illuminate/container": "^12.35",
         "nette/utils": "^4.0.7",
-        "sebastian/diff": "^6.0.2",
+        "sebastian/diff": "^6.0.2|^7.0.0",
```

Comment:
please revert this, we only use php 8.2, and downgraded version is choose the installed one.